src/Compilers/Core/Portable/InternalUtilities/EnumUtilties.cs

```diff
@@ -56,11 +56,11 @@ internal static ulong ConvertEnumUnderlyingTypeToUInt64(object value, SpecialTyp
         }
 
 #if DEBUG
-        internal static bool ContainsAllValues<T>(int mask) where T : struct, Enum, IConvertible
+        internal static bool ContainsAllValues<T>(long mask) where T : struct, Enum, IConvertible
```

Comment:
>long [](http://example.com/codeflow?start=50&length=4)

Consider adding an overload instead. Conversion is actually going to affect bits and it is probably better to not take chances even if there is nothing obviously wrong in the code.